diff --git a/php/templates/containers.twig b/php/templates/containers.twig
index cd8e73e8..fdf90a2a 100644
--- a/php/templates/containers.twig
+++ b/php/templates/containers.twig
@@ -87,14 +87,6 @@
                     {% if isDomaincheckRunning == false %}
                         <h2>Domaincheck container is not running</h2>
                         <p>This is not expected. Most likely this happened because port {{ apache_port }} is already in use on your server. You can check the mastercontainer logs and domaincheck container logs for further clues. You should be able to resolve this by adjusting the APACHE_PORT by following the <strong><a target="_blank" href="https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md">reverse proxy documentation</a></strong>. Advice: have a detailed look at the changed docker run command for AIO.</p>
-                    {% elseif is_mastercontainer_update_available == true %}
-                        <h2>Mastercontainer update</h2>
-                        <p>⚠️ A mastercontainer update is available. Please click on the button below to update it. Afterwards, you will be able to proceed with the setup.</p>
-                        <form method="POST" action="api/docker/watchtower" target="overlay-log">
-                            <input type="hidden" name="{{csrf.keys.name}}" value="{{csrf.name}}">
-                            <input type="hidden" name="{{csrf.keys.value}}" value="{{csrf.value}}">
-                            <input type="submit" value="Update mastercontainer" />
-                        </form>
                     {% else %}
                         {% if not hasBackupLocation %}
                             <p>The official Nextcloud installation method. Nextcloud All-in-One provides easy deployment and maintenance with most features included in this one Nextcloud instance.</p>
@@ -357,14 +349,6 @@
                                 {% endif %}
                                 <p>Clicking on the button below will download all docker containers and start them. This can take a long time depending on your internet connection. Since the overall size is a few GB, this can take around 5-10 min or more. Please be patient!</p>
                             {% endif %}
-                            {% if is_mastercontainer_update_available == true %}
-                                <p>⚠️ A mastercontainer update is available. Please click on the button below to update it.</p>
-                                <form method="POST" action="api/docker/watchtower" target="overlay-log">
-                                    <input type="hidden" name="{{csrf.keys.name}}" value="{{csrf.name}}">
-                                    <input type="hidden" name="{{csrf.keys.value}}" value="{{csrf.value}}">
-                                    <input type="submit" value="Update mastercontainer" />
-                                </form>
-                            {% else %}
                                 {% if was_start_button_clicked == false %}
                                     <form method="POST" action="api/docker/start" target="overlay-log">
                                         <input type="hidden" name="{{csrf.keys.name}}" value="{{csrf.name}}">
@@ -396,7 +380,6 @@
                                         <input class="button " type="submit" value="Start and update containers" data-confirm='Start and update containers? You should consider creating a backup first.' />
                                     </form>
                                 {% endif %}
-                            {% endif %}
                         {% endif %}
                     {% endif %}
 
@@ -568,7 +551,7 @@
                                                     <input type="hidden" name="{{csrf.keys.name}}" value="{{csrf.name}}">
                                                     <input type="hidden" name="{{csrf.keys.value}}" value="{{csrf.value}}">
                                                     <input type="submit" value="Submit daily backup time and settings" /><br>
-                                                    <input type="checkbox" id="automatic_updates" name="automatic_updates" checked="checked"><label for="automatic_updates">Automatically update all containers, the mastercontainer and on saturdays your Nextcloud apps</label><br>
+                                                    <input type="checkbox" id="automatic_updates" name="automatic_updates"><label for="automatic_updates">Automatically update all containers, the mastercontainer and on saturdays your Nextcloud apps</label><br>
                                                     <input type="checkbox" id="success_notification" name="success_notification" checked="checked"><label for="success_notification">Send notifications about successful backups (notifications about unsuccessful backups will always be sent)</label>
                                                 </form>
                                             {% else %}
