diff --git a/php/src/Docker/DockerActionManager.php b/php/src/Docker/DockerActionManager.php
index e6ada057..2f5b3adc 100644
--- a/php/src/Docker/DockerActionManager.php
+++ b/php/src/Docker/DockerActionManager.php
@@ -785,8 +785,8 @@ readonly class DockerActionManager {
             if (count($imageNameArray) === 2) {
                 $imageName = $imageNameArray[0];
             } else {
-                error_log("Unexpected image name was found when getting the current image name of the mastercontainer. You probably did not follow the documentation correctly. Changing the image name to the default 'ghcr.io/nextcloud-releases/all-in-one'.");
-                $imageName = 'ghcr.io/nextcloud-releases/all-in-one';
+                error_log("Unexpected image name was found when getting the current image name of the mastercontainer. You probably did not follow the documentation correctly. Changing the image name to the default 'ghcr.io/aps-conecta/all-in-one'.");
+                $imageName = 'ghcr.io/aps-conecta/all-in-one';
             }
             apcu_add($cacheKey, $imageName);
             return $imageName;
@@ -1108,7 +1108,7 @@ readonly class DockerActionManager {
     }
 
     public function isNextcloudImageOutdated(): bool {
-        $createdTime = $this->GetCreatedTimeOfNextcloudImage('ghcr.io/nextcloud-releases/aio-nextcloud');
+        $createdTime = $this->GetCreatedTimeOfNextcloudImage('ghcr.io/aps-conecta/aio-nextcloud');
 
         if ($createdTime === null) {
             $createdTime = $this->GetCreatedTimeOfNextcloudImage('nextcloud/aio-nextcloud');
